.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
The red circle with the minus sign indicates that the linter wants a single new line after your final line of code. I only know this because Max pinged me on this on my last PR🤪